locale/de_DE/submission.xml

```diff
@@ -95,6 +100,10 @@
 	<message key="submission.catalogEntry.confirm">Erstelle anhand der angegebenen Metadaten einen neuen Katalogeintrag.</message>
 	<message key="submission.catalogEntry.confirm.required">Bitte bestätigen Sie, dass der Publikationsprozess abgeschlossen ist und das Buch in den Katalog aufgenommen werden soll.</message>
 	<message key="submission.catalogEntry.isAvailable">Diese Monografie ist fertig und soll in den öffentlichen Katalog aufgenommen werden.</message>
+	<message key="submission.catalogEntry.viewSubmission">View Submission</message>
+	<message key="submission.catalogEntry.chapterPublicationDates">Publication Dates</message>
+	<message key="submission.catalogEntry.disableChapterPublicationDates">All chapters will use the publication date of the monograph.</message>
+	<message key="submission.catalogEntry.enableChapterPublicationDates">Each chapter may have its own publication date.</message>
```

Comment:
And again here.

Reply:
This commit deletes them https://github.com/pkp/omp/pull/687/commits/98ac16a8847c1d4d1bbf2c5f3c4c32c75aadaf92